planning/MartenV4.md

```diff
@@ -0,0 +1,114 @@
+# Marten v4 Ideas
+
+## Pulling it Off
+
+We've got the typical problems of needing to address incoming pull requests and bug issues in master while probably needing to have a long lived branch for *v4*. 
+
+Here are some thoughts:
+
+* First do a bug sweep 3.12 release to address as many of the tactical problems as we can before branching to v4?
```

Comment:
Yes, I think that we should do a bug bash and close as many open issues and PR that we can.